chat-plugins/mafia.js

```diff
@@ -349,7 +349,7 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (this.timer) this.setDeadline('off', true);
 		this.phase = 'night';
 		this.sendRoom(`Night ${this.dayNum}. PM the host your action, or idle.`, {declare: true});
-		if (!early && this.getPlurality()) this.sendRoom(`Plurality is on ${this.players[this.getPlurality()].name}`);
+		if (!early && this.getPlurality() && this.getPlurality() !== 'nolynch') this.sendRoom(`Plurality is on ${this.players[this.getPlurality()].name}`);
```

Comment:
It would be better to simply say that its on `No Lynch` rather than not sending a message at all.